CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## Ver 0.7.0 (24-February-2023)
+
+### New
+
+1. [END: Add function fit_discrete_mc](https://github.com/QuantEcon/QuantEcon.py/pull/681) ([jstac](https://github.com/jstac)), ([Smit-create](https://github.com/Smit-create)) and ([oyamad](https://github.com/oyamad)) which takes a time series and fits a finite markov chain
+2. [ENH: Adding Discrete Approximation of VAR Methods](https://github.com/QuantEcon/QuantEcon.py/pull/640) ([crondonm](https://github.com/crondonm)) which is a routine based on the work `Finite-State Approximation Of VAR Processes: A Simulation Approach'' by Stephanie Schmitt-Grohé and Martín Uribe, July 11, 2010.
+
+### Fixes and Maintenance
+
+1. [FIX: Fix period of reducible MarkovChain with custom state_values](https://github.com/QuantEcon/QuantEcon.py/pull/684) ([oyamad](https://github.com/oyamad))
+
+There were a range of additional maitenance and fixes including [Update __Iss.py - Remove broken link](https://github.com/QuantEcon/QuantEcon.py/pull/692), [693](https://github.com/QuantEcon/QuantEcon.py/pull/693), [689](https://github.com/QuantEcon/QuantEcon.py/pull/689), [Bring estimate_mc into top level namespace](https://github.com/QuantEcon/QuantEcon.py/pull/688), [Warn only when n in not int in tauchen](https://github.com/QuantEcon/QuantEcon.py/pull/673), [FIX: Avoid bare 'except'](https://github.com/QuantEcon/QuantEcon.py/pull/686), [FIX: DOC: Remove 2-byte spaces](https://github.com/QuantEcon/QuantEcon.py/pull/685), [MAINT: Import from _graph_tools, _gridtools](https://github.com/QuantEcon/QuantEcon.py/pull/683)
```

Comment:
This `[MAINT: Import from _graph_tools, _gridtools](https://github.com/QuantEcon/QuantEcon.py/pull/683)` is just a leftover from #682; it doesn't have to be included here.